modules/sso.php

```diff
@@ -986,9 +995,18 @@ public function error_msg_enable_two_step( $message ) {
 	 * @return string
 	 */
 	public function error_msg_email_already_exists( $message ) {
-		$err = __( sprintf( 'You already have an account on this site. Please visit your <a href="%1$s">profile page</a> page to link your account to WordPress.com!', admin_url( 'profile.php' ) ) , 'jetpack' );
+		$error = sprintf(
+			wp_kses(
+				__(
+					'You already have an account on this site. Please log in <a href="%1$s">with your username and password</a> and connect your account to WordPress.com.',
```

Comment:
You already have an account on this site. Please <a href="%1$s">sign in</a> with your username and password and then connect to WordPress.com.